### PR TITLE
Add Gatsby installation to startup docs

### PR DESCRIPTION
**Pull request by emplums**
Fixes: #903

### CODE REVIEW

**Comment by koddsson** [comment]
Could we install gatsby as a dev dependency instead?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,7 +43,13 @@ A common question asked about Primer Components is how to know what should be ad
 
 We primarily use our documentation site as a workspace to develop new components or make changes to existing components (stay tuned for a better development environment coming soon!).
 
-To get the documentation site running locally run the following in your terminal:
+Before running the documentation site locally you'll need to install packages and make sure you've got the [Gatsby cli](https://www.gatsbyjs.com/docs/gatsby-cli/) installed on your machine:
+
+```sh
+yarn && yarn global add gatsby-cli
+```
```
Comment (colebemis) [inline CONTRIBUTING.md:50]:
```suggestion
Before running the documentation site locally, you'll need to install packages in the root and `docs` directories:

Gatsby is already a dependency in the `docs` site. I think the issue that caused #903 is that the `docs` dependencies weren't installed.
```shell
yarn && cd docs; yarn 
```
```

**Review by colebemis — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,7 +43,12 @@ A common question asked about Primer Components is how to know what should be ad
 
 We primarily use our documentation site as a workspace to develop new components or make changes to existing components (stay tuned for a better development environment coming soon!).
 
-To get the documentation site running locally run the following in your terminal:
+Before running the documentation site locally, you'll need to install packages in the root and `docs` directories:
+
+```shell
+yarn && cd docs; yarn
```
Comment (emplums) [inline CONTRIBUTING.md:49]:
```suggestion
yarn && cd docs && yarn 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,7 +43,12 @@ A common question asked about Primer Components is how to know what should be ad
 
 We primarily use our documentation site as a workspace to develop new components or make changes to existing components (stay tuned for a better development environment coming soon!).
 
-To get the documentation site running locally run the following in your terminal:
+Before running the documentation site locally, you'll need to install packages in the root and `docs` directories:
+
+```shell
+yarn && cd docs; yarn 
+
+Then run the following to start up the site:
```
Comment (emplums) [inline CONTRIBUTING.md:51]:
```suggestion
Then navgiate back to the root folder and run the following to start up the site:
```

**Pull request merged**